backend/tests/check_email.rs

```diff
@@ -104,3 +104,36 @@ async fn test_reacher_secret_correct_secret() {
 	assert_eq!(resp.status(), StatusCode::OK, "{:?}", resp.body());
 	assert_eq!(resp.body(), FOO_BAR_RESPONSE);
 }
+
+#[tokio::test]
+async fn test_reacher_to_mail_empty() {
+	env::set_var("RCH_HEADER_SECRET", "foobar");
+
```

Comment:
```suggestion
```

No need to set RCH_HEADER_SECRET for this test

Reply:
If I omit the header bits the test fails with error "Missing request header"